views/tos.ejs

```diff
@@ -18,6 +18,9 @@
     <ul>
         <li>You allow your IP to be logged whilst making an account (This is so we can verify you're not
             multi-accounting)</li>
+        <li>You may not paste anything containing personal information, if you want to share personal info don't do it
+            here. This includes doxxes.</li>
+        <li>You may not abuse our api.</li>
```

Comment:
thank you

Reply:
also make api all caps

Reply:
May we correct this to be `API` (capitalized)

Reply:
May we correct this to be API (capitalized)

Reply:
Alr will do next push